lib/carrierwave/sanitized_file.rb

```diff
@@ -185,6 +185,7 @@ def move_to(new_path, permissions=nil, directory_permissions=nil, keep_filename=
       mkdir!(new_path, directory_permissions)
       move!(new_path)
       chmod!(new_path, permissions)
+      self.content_type = ::MIME::Types.type_for(new_path).first.to_s
```

Comment:
Setting content_type in `move_to` is not intuitive, just moving file usually doesn't cause content-type change.

How about setting content_type in `manipulate!` ?
https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/processing/mini_magick.rb#L295
https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/processing/rmagick.rb#L341